rails/build_docker_image.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Rails Releases
+
+# Build base image
+cd rails_releases/rails_benchmarks
+sudo docker build -t rubybench/rails_releases .
+
+# Rails Trunk
+
+# Build base image
+cd ../../rails_trunk/rails_benchmarks
+sudo docker build -t rubybench/rails_trunk .
```

Comment:
Please split the build script into the right folders. We don't want to be build two images even if we only need one